### ISSUE TITLE
After compilation, only 'libtensorflow_cc.so' was generated, and 'liblibtensorflow_cc.so.ifso' is missing

### ISSUE THREAD

**Issue by zengtianzeng** [tensorflow/tensorflow#62111]
### Issue type

Build/Install

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

source

### TensorFlow version

2.6.0

### Custom code

Yes

### OS platform and distribution

Windows

### Python version

3.7.16

### Bazel version

3.7.2

### GCC/compiler version

2019

### CUDA/cuDNN version

11.2/8.1

### GPU model and memory

3090

### Current behavior?

"libtensorflow_cc.so"和"liblibtensorflow_cc.so.ifso" were both generated.

### Standalone code to reproduce the issue

```shell
only 'libtensorflow_cc.so' was generated, and 'liblibtensorflow_cc.so.ifso' is missing
```

**Comment by tilakrayal** [comment]
@zengtianzeng,
If you need .dll files, you should only rename `bazel-bin\tensorflow\libtensorflow_cc.so` to `tensorflow_cc.dll.` and `bazel-bin\tensorflow\liblibtensorflow_cc.so.ifso` to `tensorflow_cc.lib`.

Also you are using tensorflow v2.6 and bazel 3.7.2 which are pretty old. So I request to update the tensorflow for the latest stable version 2.14 from the official document.

GPU support on native-Windows is only available for 2.10 or earlier versions, starting in TF 2.11, CUDA build is not supported for Windows. For using TensorFlow GPU on Windows, you will need to build/install TensorFlow in WSL2 or use tensorflow-cpu with TensorFlow-DirectML-Plugin.
https://www.tensorflow.org/install/source_windows

Thank you!

**Comment by zengtianzeng** [comment]
> @zengtianzeng, If you need .dll files, you should only rename `bazel-bin\tensorflow\libtensorflow_cc.so` to `tensorflow_cc.dll.` and `bazel-bin\tensorflow\liblibtensorflow_cc.so.ifso` to `tensorflow_cc.lib`.
> 
> Also you are using tensorflow v2.6 and bazel 3.7.2 which are pretty old. So I request to update the tensorflow for the latest stable version 2.14 from the official document.
> 
> GPU support on native-Windows is only available for 2.10 or earlier versions, starting in TF 2.11, CUDA build is not supported for Windows. For using TensorFlow GPU on Windows, you will need to build/install TensorFlow in WSL2 or use tensorflow-cpu with TensorFlow-DirectML-Plugin. https://www.tensorflow.org/install/source_windows
> 
> Thank you!

**Comment by zengtianzeng** [comment]
> @zengtianzeng, If you need .dll files, you should only rename `bazel-bin\tensorflow\libtensorflow_cc.so` to `tensorflow_cc.dll.` and `bazel-bin\tensorflow\liblibtensorflow_cc.so.ifso` to `tensorflow_cc.lib`.
> 
> Also you are using tensorflow v2.6 and bazel 3.7.2 which are pretty old. So I request to update the tensorflow for the latest stable version 2.14 from the official document.
> 
> GPU support on native-Windows is only available for 2.10 or earlier versions, starting in TF 2.11, CUDA build is not supported for Windows. For using TensorFlow GPU on Windows, you will need to build/install TensorFlow in WSL2 or use tensorflow-cpu with TensorFlow-DirectML-Plugin. https://www.tensorflow.org/install/source_windows
> 
> Thank you!

Thank you!
